specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/getMachineLearningServicesUsages.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "scope": "subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.MachineLearningServices/locations/eastus",
+    "api-version": "2021-03-15"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
```

Comment:
> { [](http://example.com/codeflow?start=9&length=2)

Missing 'type', 'name', 'id'.

Reply:
This is a list API. 
https://docs.microsoft.com/en-us/rest/api/compute/usage/list - Compute doesn't provides name as - https://docs.microsoft.com/en-us/rest/api/compute/usage/list#usagename
Which is included for each element in the Array.
We don't know the Type, untill the RP provides us.
We can create Id, as we discussed earlier, but it it's not pointing to the real  Azure resource owned by Compute.
Please suggest?

Reply:
These are about your resource type.

[https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc016-incorrect-resource-requests-and-responses](https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc016-incorrect-resource-requests-and-responses)

Reply:
Added -  "type": "Microsoft.Quota/quotaLimits"